cypress/integration/chi-js/chi-tabs.spec.js

```diff
@@ -0,0 +1,122 @@
+describe('Chi Tabs', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/tabs.html');
+  });
+//horizontal tab
+
+describe('horizontal tab contains sliding border', () => {
+it(' horizontal tab contains sliding border', () => {
+  
+cy.get('#example-horizontal-base li:nth-child(4)').should('have.class', 'chi-sliding-border');
+});
+});
+
+describe('Clicking horizontal tab 2 shows corresponding content 2 and active class is present',()=>{
+  it('Clicking horizontal tab 2 shows corresponding content 2 and active class is present', () => {
```

Comment:
It replicates the description of the test.. Describe can be used to group the tests by context.
like:
```
describe('Clicking horizontal tab shows corresponding content',()=>{
  it('Clicking horizontal tab 2 shows corresponding content 2 and active class is present', () => {
    /*  */
  });

  it('Clicking horizontal tab 3 shows corresponding content 23 and active class is present', () => {
  /*  */
  });
});
```

Reply:
updated the describe content.